### PR TITLE
grub-file: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

It resolves part of issue #8080

### CODE REVIEW

File: pages/linux/grub-file.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Check if a file is an i386 EFI image:
+
+`grub-file --is-i386-efi {{path/to/file}}`
+
+- Check if a file is an x86_64 EFI image:
+
+`grub-file --is-x86_64-efi {{path/to/file}}`
+
+- Check if a file is an ARM image (linux kernel):
```
Comment (kbdharun) [inline pages/linux/grub-file.md:18]:
```suggestion
- Check if a file is an ARM image (Linux kernel):
```

File: pages/linux/grub-file.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`grub-file --is-arm-linux {{path/to/file}}`
+
+- Check if a file is an x86 image (linux kernel):
```
Comment (kbdharun) [inline pages/linux/grub-file.md:22]:
```suggestion
- Check if a file is an x86 image (Linux kernel):
```

File: pages/linux/grub-file.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# grub-file
+
+> Check if a file is of a specific bootable image type.
+> More information: <https://manned.org/grub-file.1>.
```
Comment (kbdharun) [inline pages/linux/grub-file.md:4]:
```suggestion
> More information: <https://manned.org/grub-file>.
```
`.1` isn't required here, as there isn't another page with same name.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**